ios/FluentUI.Demo/FluentUI.Demo/Demos/ListVnextDemoController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ListVnextDemoController: DemoController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        container.alignment = .leading
+        let sections: [TableViewSampleData.Section] = TableViewCellSampleData.sections
+        var section: TableViewCellSampleData.Section
+        var cell: TableViewCellSampleData.Item
+        let numSections = sections.count - 1
+        let numRows = TableViewCellSampleData.numberOfItemsInSection - 1
+        var indexPath = IndexPath(row: 0, section: 0)
+
+        var list: MSFListVnext
+        var listItem: MSFListVnextCellData
+        var listItems: [MSFListVnextCellData]
+        let iconStyle = MSFListIconVnextStyle.none
+
+        for sectionCount in 0...numSections {
+            section = sections[sectionCount]
+
+            listItems = []
+            for row in 0...numRows {
+                cell = section.item
+                listItem = MSFListVnextCellData()
+                listItem.title = cell.text1
+                listItem.subtitle = cell.text2
+                listItem.leadingIcon = UIImage(named: cell.image)
+                listItem.trailingIcon = accessoryType(for: row)
+                listItem.onTapAction = {
+                    indexPath.row = row
+                    indexPath.section = sectionCount
+                    self.showAlertForCellTapped(indexPath: indexPath)
+                }
+                listItems.append(listItem)
+            }
+
+            list = MSFListVnext(cells: listItems, layoutType: updateLayout(subtitle: listItems[0].subtitle), iconStyle: iconStyle)
+            list.state.sectionTitle = section.title
+            addRow(items: [list.view])
```

Comment:
If you remove line 12 and use **container.addArrangedSubview(list.view)** instead, you'll get the constraint behavior we want (taking up the entire width and height:


https://user-images.githubusercontent.com/68076145/103374696-14847000-4a8d-11eb-998a-af9b15341239.mov